Example/src/main/java/net/hypixel/example/skyblock/GetSkyBlockProfileExample.java

```diff
@@ -14,8 +13,10 @@ public static void main(String[] args) {
                 return;
             }
 
-            for (Map.Entry<String, JsonElement> profileEntry : playerReply.getPlayer().getAsJsonObject("stats").getAsJsonObject("SkyBlock").getAsJsonObject("profiles").entrySet()) {
-                ExampleUtil.API.getSkyBlockProfile(profileEntry.getKey()).whenComplete(ExampleUtil.getTestConsumer());
+            for (Map.Entry<String, JsonElement> profileEntry : playerReply.getPlayer()
+                .getObjectProperty("stats.SkyBlock.profiles").entrySet()) {
+                ExampleUtil.API.getSkyBlockProfile(profileEntry.getKey())
+                    .whenComplete(ExampleUtil.getTestConsumer());
                 break;
             }
```

Comment:
Gson didn't add keySet to objects until [2.8.1](https://github.com/google/gson/commit/4976e42), and the API was only using 2.3 when I forked. I could change this once it's merged into 4.0.0, since that seems to be using a newer Gson version.

I do agree that the example is confusing though, and I'll be sure to clean it up.